Gir.toml

```diff
@@ -1535,7 +1535,7 @@ status = "generate"
 [[object]]
 name = "Gtk.Plug"
 status = "generate"
-cfg_condition = "all(not(windows),not(target_os = \"macos\"))"
+cfg_condition = "target = \"x11\""
```

Comment:
I didn't know you could define your own targets :open_mouth: 

We should also do the same with `gio-unix` and things like that I guess.

Reply:
Not sure what you mean re `gio-unix` since this is mostly a gdk/gtk thing.

Reply:
There's a similar problem with GIO (and GLib) about Windows/POSIX/Linux specific APIs.

Reply:
Aah, I don't think gio exports such info in their pkg-config but perhaps there's a different way to discover it for that.

Reply:
Separate `pkg-config` files